### DISCUSSION TITLE
Future posts sudden disappearance

### DISCUSSION THREAD

**Discussion by LWVPullman** [mmistakes/minimal-mistakes#3406] [Q&A] [answered]
My non-profit organization is using minimal mistakes to build our website. We post future events into the blog platform. All of the sudden, future posts have disappeared. Ones that we used to be able to see on this page, are now missing https://lwvpullman.org/year-archive/ . Has something changed recently with github or minimal mistakes that would make this happen?

**Comment by mmistakes** [comment, marked as answer]
No changes to the theme that would impact future posts. Not sure if GitHub Pages changed something, but typically you need to enable it with `future: true` in your _config.yml or with a flag e.g., `jekyll build --future`

**Reply by LWVPullman** [reply]
huh, I don't see anything in the config.yml that says "future". Guess I'll add it. Perhaps it could be added into the current template? It's weird that future worked just 2 weeks ago, and then suddenly isn't working anymore.

**Comment by mmistakes** [comment]
Yeah I dunno. This is one of those "Jekyll configuration" things and has nothing to do with the theme. By default Jekyll doesn't build posts or collections that are future dated.

Entirely possible this default behavior changed in Jekyll if GitHub Pages updated what version they support. Or they were previously setting `future: true` and now aren't. Who knows, I don't follow the changes they make closely enough to know if something is different.

If you build your site locally you'll know for sure what's going on. If you don't have `future: true` you'll see something like the following, which will clue you in if you need to enable it or not.

```
Skipping: _posts/2999-your-future-dated-post.md
```

**Reply by LWVPullman** [reply]
Seems to have fixed it :) Thanks!
